internal/metrics/smbstatus_test.go

```diff
@@ -602,3 +616,16 @@ func TestParseSMBStatusLocks(t *testing.T) {
 	assert.Equal(t, lock2.FileID.Inode, int64(52))
 	assert.Equal(t, lock2.NumPendingDeletes, 2)
 }
+
+func TestParseSMBStatusLease(t *testing.T) {
```

Comment:
May be add a test for `oplocks` too?

Reply:
You are right -- I should have done so in the first place..

Reply:
Done